docs/pages/release_notes.rst

```diff
@@ -9,14 +9,16 @@ Peanut (v6.0.0)
 **Release date:** TBD
 **Full Changelog**: https://github.com/nuts-foundation/nuts-node/compare/v5.0.0...v6.0.0
 
-Breaking
-********
```

Comment:
I think (based on the rst2myst conversion results) that this puts it at the same level as
```
*******************
Peanut (v6.0.0)
*******************
```
(too many *** by the way)

At the bottom of this file we sort of specified how add subsections.

Reply:
no `=` vs `*`